### PR TITLE
Exact validation computation in TensorFlow

### PR DESCRIPTION
**Pull request by alecgunny**
Changing `KerasSequenceValidater` to leverage metric `update_state` method in order to do exact compute of global metrics.

### CODE REVIEW

**Comment by alecgunny** [comment]
Addresses #405

**Review by benfred — APPROVED** [review]
